tools/cli/test/data/changelog/new-api-preview-version/base/metadata.json

```
@@ -0,0 +1 @@
+{"runDate": "2024-08-07", "runId": "20240807085043", "specRevision": "63cbfec3f6eeceb446c1d2641263b0628f04f450", "specRevisionShort": "63cbfec3f6e", "versions": ["2023-01-01", "2023-02-01", "2023-10-01", "2023-11-15", "2024-05-30"], "migration": 1}
```

```suggestion
{"runDate": "2024-08-07", "runId": "20240807085043", "specRevision": "63cbfec3f6eeceb446c1d2641263b0628f04f450", "specRevisionShort": "63cbfec3f6e", "versions": ["2023-01-01", "2023-02-01", "2023-10-01", "2023-11-15", "2024-05-30"]}
```
Migration is not supported (we do it manually if needed)